charts/dgrsc/values.yaml

```diff
@@ -0,0 +1,133 @@
+      stabilizationWindowSeconds: 120
+      selectPolicy: Max
+      policies:
+        - type: Pods
+          value: 1
+          periodSeconds: 120
+    scaleUp:
+      stabilizationWindowSeconds: 0
+      selectPolicy: Max
+      policies:
+        - type: Pods
+          value: 2
+          periodSeconds: 30
+  minReplicas: 2
+  maxReplicas: 16
+  targetCPUUtilizationPercentage: 80
+  # targetMemoryUtilizationPercentage: 80
+
+serviceAccount:
+  # Specifies whether a service account should be created
+  enabled: false
+  # Annotations to add to the service account
+  annotations: {}
+  # The name of the service account to use.
+  # If not set and create is true, a name is generated using the fullname template
+  name: ~
+
+networkPolicy:
+  enabled: true
+
+service:
+  type: ClusterIP
+  port: 8080
+  portName: http
+
+route:
+  enabled: true
+  annotations: {}
+    # kubernetes.io/ingress.class: nginx
+    # kubernetes.io/tls-acme: "true"
+  host: chart-example.local
+  path: /
+  tls:
+    insecureEdgeTerminationPolicy: Redirect
+    termination: edge
+  wildcardPolicy: None
+
+resources:
+  # We usually recommend not to specify default resources and to leave this as a conscious
+  # choice for the user. This also increases chances charts run on environments with little
+  # resources, such as Minikube. If you do want to specify resources, uncomment the following
+  # lines, adjust them as necessary, and remove the curly braces after 'resources:'.
+  limits:
+    cpu: 200m
+    memory: 256Mi
+  requests:
+    cpu: 10m
+    memory: 128Mi
+
+config:
+  # Set to true if you want to let Helm manage and overwrite your configmaps.
+  enabled: false
+
+  # This should be set to true if and only if you require configmaps and secrets to be release
+  # scoped. In the event you want all instances in the same namespace to share a similar
+  # configuration, this should be set to false
+  releaseScoped: false
+
+  # These values will be wholesale added to the configmap as is; refer to the dgrsc
+  # documentation for what each of these values mean and whether you need them defined.
+  # Ensure that all values are represented explicitly as strings, as non-string values will
+  # not translate over as expected into container environment variables.
+  # For configuration keys named `*_ENABLED`, either leave them commented/undefined, or set them
+  # to string value "true".
+  configMap:
+    # FRONTEND_KC_CLIENTID: ~
+    # FRONTEND_KC_REALM: ~
+    # FRONTEND_KC_SERVERURL: ~
+    FRONTEND_APIPATH: "api/v2"
+    FRONTEND_BASEPATH: "/app"
+    # FRONTEND_DASHBOARDURL: ~
+    # SERVER_KC_CLIENTID: ~
+    # SERVER_KC_PUBLICKEY: ~
+    # SERVER_KC_REALM: ~
+    # SERVER_KC_SERVERURL: ~
+    SERVER_APIPATH: "/api/v2"
+    SERVER_BASEPATH: "/app"
+    SERVER_BODYLIMIT: "30mb"
+    # SERVER_HOSTURL: ~
+    # SERVER_LOGFILE: ~
+    SERVER_LOGLEVEL: "http"
+    SERVER_PORT: "8080"
+    # CDOGS_APIURL: ~
+    # CDOGS_CLIENTID: ~
+    # CDOGS_TOKENURL: ~
```

Comment:
These are commented meaning they are optional in configMap setup. We shouldn't be having the username/clientId side of the secret being publicly visible on a github repository.